solr/solr-ref-guide/src/nested-documents.adoc

```diff
@@ -0,0 +1,321 @@
+= Nested Child Documents
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+Solr supports indexing nested documents such as a blog post parent document and comments as child documents -- or products as parent documents and sizes, colors, or other variations as child documents.
+The parent with all children is referred to as a "block" and it explains some of the nomenclature of related features.
```

Comment:
This paragraph should be split up a little bit. In our CSS the first paragraph of any page in the HTML format is presented in a slightly larger font - the way this is written a large block of the page is presented larger. Simply adding another line break between the first sentence and the next sentences will allow the first sentence to be presented in that larger format and the rest in the regular page font. Doing this will also make the note that follows this paragraph to not look as odd as it does.